tests/test_metadata_utils.py

```diff
@@ -359,1254 +364,32 @@ def test_add_metadata_and_chunk_examples(self):
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-            ],
-        )
-        self.assertEqual(
-            mapped_ds[0]["metadata_mask"],
-            [
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                1,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-                0,
-            ],
-        )
+        # fmt: off
+        self.assertEqual(self.tokenizer.convert_ids_to_tokens(mapped_ds[0]["input_ids"]), ["url", ":", "Ġhttps", "://", "www", ".", "bb", "c", ".", "com", "/", "s", "port", "/", "live", "/", "oly", "mp", "ics", "/", "509", "74", "152", "Ġ|", "Ġtimestamp", ":", "Ġ2018", "-", "12", "-", "10", "T", "13", ":", "45", ":", "00", ".", "000", "Z", "Ġ||", "|", "ĠIt", "Ġwas", "Ġa", "Ġbrilliant", "Ġfirst", "Ġround", ".", "ĠYou", "Ġhave", "Ġto", "Ġbreak", "Ġdown", "Ġthe", "ĠCuban", "'s", "Ġrhythm", "Ġyou", "Ġcan", "'t", "Ġlet", "Ġthem", "Ġget"])
+        self.assertEqual(mapped_ds[0]["attention_mask"], [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1])
+        self.assertEqual(mapped_ds[0]["metadata_mask"], [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ])
```

Comment:
I reformatted the tests as the vertical format made it complicated to navigate / update them!